src/contracts/Repository/ContentService.php

```diff
@@ -151,16 +131,13 @@ public function loadContentByVersionInfo(VersionInfo $versionInfo, array $langua
      *
      * If no version number is given, the method returns the current version
      *
-     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException if the content or version with the given id and languages does not exist
-     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException If the user has no access to read content and in case of un-published content: read versions
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException if the content or version with the given id and languages doesn't exist.
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException if the user has no access to "read content", or, in case of un-published content, "read versions".
```

Comment:
```suggestion
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException if the user has no access to "read content" or in case of unpublished content - "read versions".
```

Reply:
@konradoboza I'm a bit surprised by this usage of dashes. I'll check if it's regular grammar.

Reply:
I am not 100% sure if it's better. It's not a biggie to me tbh.

Reply:
```suggestion
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException if the user has no access to "read content", or, in case of unpublished content, "read versions".
```

Reply:
I'm comming back to this. I think that it should also use the module/fundtion format with the actual names.

```suggestion
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException if the user has no access to "content/read", or, in case of unpublished content, "content/versionread".
```

Reply:
And now for something ~~completely~~  different:
```suggestion
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException if the user lacks:
     *                         - `content/read` permission for published content, or
     *                         - `content/read` and `content/versionread` permissions for draft content.
```

Thank you @mnocon for the discussion leading to this formulation.

@konradoboza Isn't this last proposal better?

Reply:
That works.